### DISCUSSION TITLE
Possibility to change favicon?

### DISCUSSION THREAD

**Discussion by XHyperDEVX** [shlinkio/discussions#21] [Help wanted] [answered]
### shlink-web-client version

v3.10.2

### How do you use shlink-web-client

Docker image

### Summary

Is it possible to change the favicon? So that it remains even after an image update?

**Comment by acelaya** [comment, marked as answer]
Main favicons are located in the [public](https://github.com/shlinkio/shlink-web-client/tree/2f3b4104b96e72b67efa7eaceb3e683f686b7099/public) directory, so you could overwrite them via docker volumes, or by creating your own image based on shlink-web-client one.

You would also have to take into consideration device-specific icons from [public/icons](https://github.com/shlinkio/shlink-web-client/tree/2f3b4104b96e72b67efa7eaceb3e683f686b7099/public/icons) directory, but there are online favicon generator tools. Just Google for "generate favicon".

**Reply by XHyperDEVX** [reply]
Thank you!

**Reply by XHyperDEVX** [reply]
@acelaya I have now changed the favicon.ico there. However, it is not adopted in the self-hosted web client. Have I done something wrong?

**Reply by XHyperDEVX** [reply]
@acelaya 
I have now changed the favicon. I have replaced all icons, including those in the "icons" subfolder. Only the icons on the page itself have not changed. Are the icons located somewhere else?
